networkit/cpp/graph/test/GraphToolsGTest.cpp

```diff
@@ -1041,6 +1041,34 @@ TEST_P(GraphToolsGTest, testEdgesSortedByWeight) {
     }
 }
 
+TEST_P(GraphToolsGTest, testParallelSumEdgesIndexed) {
```

Comment:
TEST_P already tests for combinations of weighted/unweighted + directed/undirected. Better use this syntax (removes the duplicated code). Also, currently this test code should be launched four times (for all combinations of the parametrized test).